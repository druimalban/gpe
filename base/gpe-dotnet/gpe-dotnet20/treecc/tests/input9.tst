// literal code sections
{ abc^*&%/* kjk}jkj*/
	// }
	/*
*/
	"\"}" '/*kjkjk*/'
{ abc // }
}
}
{ // open block
