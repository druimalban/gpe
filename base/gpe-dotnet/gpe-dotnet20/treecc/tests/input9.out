line 2: literal code:
 abc^*&%/* kjk}jkj*/
	// }
	/*
*/
	"\"}" '/*kjkjk*/'
{ abc // }
}

input9.tst:10: end of file inside literal code block
line 10: literal code:
 // open block

